### ISSUE TITLE
Check Not Run When Pull Request Opened

### ISSUE THREAD

**Issue by Kurt-von-Laven** [dependabot/dependabot-core#5832]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

npm

### dependabot.yml content

[`.github/dependabot.yaml`](https://github.com/ScribeMD/docker-cache/blob/4944442746672372ffe1c972d27473bc52fbcbe7/.github/dependabot.yaml)

### What you expected to see, versus what you actually saw

[No Dependabot check was run](https://github.com/ScribeMD/docker-cache/actions/runs/3151749009) when ScribeMD/docker-cache#157 was opened proposing an incorrect change to `.github/dependabot.yaml`. [The Dependabot check](https://github.com/ScribeMD/docker-cache/runs/8620330664) was only run after the pull request was merged to our default branch, `main`. That is later than would be ideal since a separate pull request had to be opened to fix the issue found by the check:

```
The property '#/updates/2' is a duplicate. Update configs must have a unique combination of 'package-ecosystem', 'directory', and 'target-branch'
```

### Images of the diff or a link to the PR, issue, or logs

ScribeMD/docker-cache#157

**Comment by jeffwidman** [comment]
👋 Unfortunately this is a known problem, see https://github.com/dependabot/dependabot-core/issues/4605

And [the fix is a bit complicated](https://github.com/dependabot/dependabot-core/issues/4605#issuecomment-1023250637).

It's still on our radar, and some of the initial work was completed to unblock this, but there's still a ways to go. For now I'm closing as a dupe of #4605
